ts/features/it-wallet/saga/itwRpPresentationSaga.ts

```diff
@@ -25,61 +32,73 @@ export function* handleItwRpPresentationSaga(
   _: ActionType<typeof itwRpPresentation.request>
 ): SagaIterator {
   try {
-    // TODO: this claims should be selected by user
-    const claims = [
-      "unique_id",
-      "given_name",
-      "family_name",
-      "birthdate",
-      "place_of_birth",
-      "tax_id_number",
-      "evidence"
-    ];
-
-    const requestObjectValue = yield* select(
-      itwRpInitializationRequestObjectValueSelector
+    yield* put(
+      identificationRequest(false, true, undefined, {
+        label: I18n.t("global.buttons.cancel"),
+        onCancel: () =>
+          NavigationService.dispatchNavigationAction(CommonActions.goBack())
+      })
```

Comment:
What do you think to use verifyPin()?

Reply:
This one slipped away, thank you. Addressed in bdbf0f0.